spec/lib/query/query_payments_spec.rb

```diff
@@ -790,6 +790,29 @@ def generate_donation(h)
         expect(result[:data].count).to eq 4
         expect(result[:data]).to_not satisfy { |i| i.any? { |j| j["id"] == donation_result_tomorrow["payment"]["id"] } }
       end
+
+      context 'when filtering by campaign' do
+        it 'returns 2 campaign results' do 
+          donation_result_today
+          donation_result_yesterday
+          donation_result_tomorrow
+
+          result = QueryPayments::full_search(nonprofit.id, {campaign_id: campaign.id})
+          expect(result[:data].count).to eq 2
+        end 
+      end 
+
+      context 'and sorts results' do 
+        it 'returns sorted campaign results' do 
+          donation_result_today
+          donation_result_tomorrow
+          donation_result_yesterday
+
+          result = QueryPayments::full_search(nonprofit.id, {campaign_id: campaign.id}) 
```

Comment:
@camckin10 how is this testing that the results from the database are sorted? Could you walk me through that?